types/src/model/mining.rs

```diff
@@ -115,6 +116,8 @@ pub struct GetMiningInfo {
     pub network_hash_ps: i64,
     /// The size of the mempool.
     pub pooled_tx: i64,
+    /// Minimum feerate of packages selected for block inclusion in BTC/kvB.
```

Comment:
```suggestion
    /// Minimum feerate of packages selected for block inclusion.
```

`FeeRate` abstracts away the unit.